client/my-sites/domains/components/domain-warnings/index.jsx

```diff
@@ -67,7 +65,6 @@ export class DomainWarnings extends PureComponent {
 			'expiredDomainsCanManage',
 			'expiringDomainsCanManage',
 			'unverifiedDomainsCanManage',
-			'pendingGSuiteTosAcceptanceDomains',
```

Comment:
There are two places where we need to remove that rule (an [example](https://github.com/Automattic/wp-calypso/blob/fc1c2034600e49a8dbe0172d9894adac6b766a4b/client/my-sites/domains/domain-management/list/index.jsx#L93)).